src/js/components/TextInput/stories/Placeholder.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Form, TextInput, Text } from 'grommet';
+
+const Placeholder = () => {
+  return (
+    <Box>
+      <Form>
+        <TextInput name="name" placeholder={<Text>footer</Text>} />
+        <TextInput name="name" placeholder="foobar" />
```

Comment:
Also, you could delete this example :thinking: , letting just the first one

Reply:
u right, focus on styled placeholders.

tnks!!